app/src/main/java/com/github/se/eduverse/model/folder/Folder.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.se.eduverse.model.folder
+
+data class Folder(
```

Comment:
I think that it's a good idea add a filterTypes property so that each folder can have its own default way of displaying the files saved from the last user choice
Also I think the folder should have a userId property to tie the folders to their owners

Reply:
Good Idea